polaris-react/src/components/Filters/Filters.scss

```diff
@@ -78,9 +78,9 @@
 }
 
 .AddFilter {
-  background: var(--p-color-bg-subdued);
-  border-radius: var(--p-border-radius-6);
-  border: var(--p-color-border-subdued) dashed var(--p-border-width-1);
+  background: var(--p-color-bg);
```

Comment:
change this to color-bg token instead to match [Figma](https://www.figma.com/file/jLLkmo9r28hiqPvf4s90E4/Polaris-Gen-3-Components-%5Balpha%5D?node-id=44707%3A19512&mode=dev)